docs/docs/overview/docker-deploy.md

```diff
@@ -95,6 +95,18 @@ When using the IFrame embed to attach the notification center rather than the Re
 
 See [Use your own backend and socket URL](https://docs.novu.co/notification-center/react-components#use-your-own-backend-and-socket-url).
 
+### Caching
```

Comment:
I was not sure about the title here.

Reply:
I think it is fine for the moment, this is usually just a baked-in feature and not really shown or talked about that often.